test/cli/test_cli.py

```diff
@@ -727,6 +727,27 @@ def test_should_call_cli_build_commands_with_NOW_and_ALL(
             export_image_tar=None,
         )
 
+    @mock.patch('bigflow.build.operate.build_project')
+    @mock.patch('bigflow.build.spec.read_project_spec')
+    @freezegun.freeze_time("2001-02-03 15:00:00")
+    def test_should_call_cli_build_with_additional_params(
+        self,
+        read_project_spec: mock.Mock,
+        build_project_mock: mock.Mock,
+    ):
+        # when
+        cli(['build','--workflow', 'ALL', '--start-time', 'NOW', '--additional-param', 'param'])
```

Comment:
Both test look almost the same, despite different arguments being used. WDYT about using `@pytest.mark.parametrize`?

Reply:
I actually did that accidentally, look now